keystone_api/apps/users/models.py

```diff
@@ -27,9 +27,9 @@ class User(auth_models.AbstractBaseUser, auth_models.PermissionsMixin):
     # User metadata
     username = models.CharField('username', max_length=150, unique=True, validators=[UnicodeUsernameValidator()])
     password = models.CharField('password', max_length=128)
```

Comment:
@djperrefort Now that password is in the model, do I need to make it nullable as well? I only provide `username` in my new user POSTs at the moment...

Reply:
I would leave the password field as required. There are other workarounds to creating a user without knowing the password (like using a temporary or invalid password).

Reply:
Would it be OK to make them nullable but not blank? I could randomly generate something, if the LDAP sync will just overwrite it, but I'm trying to hardcoding something into the requests for the password.